genesyscloud/external_contacts_organization/resource_genesyscloud_externalcontacts_organization_utils.go

```diff
@@ -0,0 +1,407 @@
+func flattenSdkAddress(address *platformclientv2.Contactaddress) []interface{} {
+	addressInterface := make(map[string]interface{})
+
+	resourcedata.SetMapValueIfNotNil(addressInterface, "address1", address.Address1)
+	resourcedata.SetMapValueIfNotNil(addressInterface, "address2", address.Address2)
+	resourcedata.SetMapValueIfNotNil(addressInterface, "city", address.City)
+	resourcedata.SetMapValueIfNotNil(addressInterface, "state", address.State)
+	resourcedata.SetMapValueIfNotNil(addressInterface, "postal_code", address.PostalCode)
+	resourcedata.SetMapValueIfNotNil(addressInterface, "country_code", address.CountryCode)
+
+	return []interface{}{addressInterface}
+}
+
+// buildSdkTwitterid maps data from a Terraform data object into a Genesys Cloud *platformclientv2.Twitterid
+func buildSdkTwitterId(d *schema.ResourceData, key string) *platformclientv2.Twitterid {
+	if d.Get(key) != nil {
+		twitterData := d.Get(key).([]interface{})
+		if len(twitterData) == 0 {
+			return nil
+		}
+		twitterMap := twitterData[0].(map[string]interface{})
+		id := twitterMap["twitter_id"].(string)
+		name := twitterMap["name"].(string)
+		screenname := twitterMap["screen_name"].(string)
+
+		return &platformclientv2.Twitterid{
+			Id:         &id,
+			Name:       &name,
+			ScreenName: &screenname,
+		}
+
+	}
+	return nil
+}
+
+// buildTickers maps an []interface{} into a Genesys Cloud *[]platformclientv2.Ticker
+func buildTickers(tickers []interface{}) *[]platformclientv2.Ticker {
+	if len(tickers) == 0 {
+		return nil
+	}
+	tickersSlice := make([]platformclientv2.Ticker, 0)
+	for _, ticker := range tickers {
+		var sdkTicker platformclientv2.Ticker
+		tickersMap, ok := ticker.(map[string]interface{})
+		if !ok {
+			continue
+		}
+
+		resourcedata.BuildSDKStringValueIfNotNil(&sdkTicker.Symbol, tickersMap, "symbol")
+		resourcedata.BuildSDKStringValueIfNotNil(&sdkTicker.Exchange, tickersMap, "exchange")
+
+		tickersSlice = append(tickersSlice, sdkTicker)
+	}
+
+	return &tickersSlice
+}
+
+// buildTrustors maps an []interface{} into a Genesys Cloud *[]platformclientv2.Trustor
+func buildTrustor(trustor []interface{}) *platformclientv2.Trustor {
+	if len(trustor) == 0 {
+		return nil
+	}
+	var sdkTrustor platformclientv2.Trustor
+
+	trustorsMap := trustor[0].(map[string]interface{})
+	sdkTrustor.Enabled = platformclientv2.Bool(trustorsMap["enabled"].(bool))
+
+	return &sdkTrustor
+}
+
+func BuildOrganizationSchema(schemaList []interface{}) (*platformclientv2.Dataschema, error) {
+	if len(schemaList) > 0 {
+		schemaMap := schemaList[0].(map[string]interface{})
+		dataSchema := &platformclientv2.Dataschema{}
+		dataSchema.Name = platformclientv2.String(schemaMap["name"].(string))
+		dataSchema.Enabled = platformclientv2.Bool(schemaMap["enabled"].(bool))
+		version := platformclientv2.Int(schemaMap["version"].(int))
+		if *version != 0 {
+			dataSchema.Version = version
```

Comment:
are we using any default schema ?

Reply:
There is no default schema but the user is allowed to use a particular version of a schema by pointing to its version & Id.